<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>charge_stationClass</class>
 <widget class="QMainWindow" name="charge_stationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>809</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>charging_station</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>563</width>
         <height>736</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="configureShowDevices">
             <property name="text">
              <string>Configure</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Firmware</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="1">
            <widget class="QPushButton" name="readImage">
             <property name="text">
              <string>Read Image</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Display</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="logToFile">
            <property name="text">
             <string>To File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QPlainTextEdit" name="logs">
          <property name="documentTitle">
           <string>Logs</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Display</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>35</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="toolButtonBeacon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Beacon</string>
            </property>
            <property name="iconSize">
             <size>
              <width>200</width>
              <height>128</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonIconOnly</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonTimekeeper">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Timekeeper</string>
            </property>
            <property name="iconSize">
             <size>
              <width>200</width>
              <height>128</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonRouter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Router</string>
            </property>
            <property name="iconSize">
             <size>
              <width>200</width>
              <height>128</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="beacon">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="1">
            <widget class="QGroupBox" name="beaconGroupBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Beacon properties</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="3" column="1">
               <widget class="QLineEdit" name="bcBeaconID">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Beacon ID:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Product ID:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="bcProductID">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Serial Number:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="bcSerialNumber">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Pan ID:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="bcPanID">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>LED On Offset:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="bcLedOnOffset">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>LED Off Offset:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="bcLedOffOffset">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Wireless Channel:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Wireless Time Slot:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>LED ID 1:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QLineEdit" name="bcLed1">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>LED ID 2:</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QLineEdit" name="bcLed2">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>LED ID 3:</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QLineEdit" name="bcLed3">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Transmit Power Level:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="bcWirelessChannel">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLineEdit" name="bcWirelessTimeSlot">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QLineEdit" name="bcTxPowerLevel">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTableView" name="beaconTableView">
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>5</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="timekeeper">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QGroupBox" name="beaconGroupBox_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Timekeeper properties</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="1" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Serial Number:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="tkProductId">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Product ID:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="tkSerialNumber">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Pan ID:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="tkPanId">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_22">
                <property name="text">
                 <string>Wireless Channel:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="tkWirelessDropDown">
                <item>
                 <property name="text">
                  <string>9</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>10</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>11</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>12</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>13</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>14</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>16</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>17</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>18</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>21</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Transmit Power Level:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="tkPowerLevel">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QPushButton" name="tkSetWirelessChannel">
                <property name="text">
                 <string>Set</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QTableView" name="timekeepeTableView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTableView" name="routerTableView"/>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="beaconGroupBox_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Router properties</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Product ID:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="rtProductId"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Serial Number:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="rtSerialNumber"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Pan ID:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="rtPanId"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_23">
                <property name="text">
                 <string>Wireless Channel:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_25">
                <property name="text">
                 <string>Transmit Power Level:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="rtWirelessChannel"/>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="rtPowerLevel"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="charging_station.qrc"/>
 </resources>
 <connections/>
</ui>
